pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -40,12 +40,152 @@
         <Exclude>Compile</Exclude>
       </Dependency>
 
+      <!-- Only need the placeholder in the netcoreapp1.0 folder -->
       <File Include="$(PlaceholderFile)" >
         <SkipPackageFileCheck>true</SkipPackageFileCheck>
         <TargetPath>lib/netcoreapp1.0</TargetPath>
       </File>
     </ItemGroup>
   </Target>
 
+  <!--
+    Manually include the netcoreapp1.0 dependencies.
+    If any versions change in the release/1.0.0 branch they should
```

Comment:
We need to produce a Microsoft.NETCore.App meta-package that will support both netcoreapp1.0 and netcoreapp1.1 and in order to do that we need to have to dependency groups. The new netcoreapp1.1 dependency set and the netcoreapp1.0 dependency set which tracks the release/1.0.0 setup. Without this people will not be able to update their Microsoft.NETCore.App meta-package and still resolve for netcoreapp1.0.